src/lib/IONOS/components/SendMessageButton.svelte

```diff
@@ -1,18 +1,42 @@
 <script lang="ts">
-	import PaperPlane from '$lib/components/icons/PaperPlane.svelte';
+	import Tooltip from '$lib/components/common/Tooltip.svelte';
+	import { getContext } from 'svelte';
+	const i18n = getContext('i18n');
 
 	export let disabled = false;
 </script>
 
-<button {disabled} type="submit" class="{$$restProps.class || ''}">
-	<PaperPlane
-		className="fill-gray-400 group-disabled/button:fill-gray-400 cursor-pointer group-disabled/button:cursor-default hover:fill-ionos"
-	/>
-</button>
+<div class={$$restProps.class || ''}>
+	<Tooltip content={$i18n.t('Send message')}>
+		<button
+			id="send-message-button"
+			class="{prompt !== ''
+				? 'exos-button exos-button--primary exos-button--large-icon-only'
+				: 'text-white bg-gray-200 dark:text-gray-900 dark:bg-gray-700 disabled'} transition rounded-full p-1.5 m-0.5 self-center"
+			type="submit"
```

Comment:
question: not quite sure, what the else block is used for, if the button is not displayed without a prompt anyway

Reply:
is copy paste from original button